### PR TITLE
Feature: Can select type of lint message 

### PR DESCRIPTION
**Pull request by Pofigizm**
In my case linter-jscs has most low priority, but it always has error type which has highest priority. I fix it. Now you can chose type of message (priority).

### CODE REVIEW

File: lib/linter-jscs.coffee [hunk 1]
```diff
@@ -27,6 +27,9 @@ class LinterJscs extends Linter
   # A string to indicate using jscs config
   config: ''
 
+  # A string to setup type of message (don't understand why default isn't work)
+  defaultLevel: atom.config.get "linter-jscs.messageType" or 'info'
```
Comment (iam4x) [inline lib/linter-jscs.coffee]: Use single quotes on `"linter-jscs.messageType"`

**Comment by iam4x** [comment]
Hello @Pofigizm 

Thank's for contribution, after the code style change I'm okay to merge :+1:

**Comment by Pofigizm** [comment]
Hi, @iam4x.
I fix code style in my part.

But this file already has mix of quote type. I only copy-paste one string... :)

**Comment by iam4x** [comment]
Thank's I'll check the rest of the file before new release :)

**Pull request merged**